src/test/java/com/github/tomakehurst/wiremock/matching/MultiValuePatternTest.java

```diff
@@ -74,14 +83,45 @@ public void returnsNonZeroDistanceWhenHeaderValuesAreSimilar() {
   }
 
   @Test
-  public void returnsTheBestMatchWhenSeveralValuesAreAvailableAndNoneAreExact() {
+  public void returnsTheBestMatchWhenSeveralHeaderValuesAreAvailableAndNoneAreExact() {
     assertThat(
         MultiValuePattern.of(equalTo("required-value"))
             .match(httpHeader("any-key", "require1234567", "requi12345", "1234567rrrr"))
             .getDistance(),
         is(0.5));
   }
 
+  @Test
+  public void returnsTheBestMatchWhenSeveralHeaderDateTimeValuesAreAvailableAndNoneAreExact() {
+    String[] actualDateTimesOverOneMinuteDiff = {
+      ACTUAL_DATE_TIME_ONE_DAY_DIFFERENCE,
+      ACTUAL_DATE_TIME_ONE_MONTH_DIFFERENCE,
+      ACTUAL_DATE_TIME_ONE_YEAR_DIFFERENCE,
+      ACTUAL_DATE_TIME_ONE_CENTURY_DIFFERENCE
+    };
+    ;
+    assertThat(
```

Comment:
This test is quite hard to read. I suggest pulling out some well-named variables for the expected and actual values in this assertion.

Reply:
Addressed in amended commit.